deployments/tyk/docker-compose.yml

```diff
@@ -49,7 +49,7 @@ services:
     depends_on:
       - tyk-redis
   tyk-gateway-2:
-    image: tykio/tyk-gateway:${GATEWAY2_VERSION:-v3.2.1}
+    image: tykio/tyk-gateway:${GATEWAY2_VERSION:-v4.0.0-rc11}
```

Comment:
Should we merge with these unstable builds or wait for official ?

Reply:
I think for merging to master we should wait for official. Happy to get things working with release candidates in the branch though.